stix_shifter_modules/onelogin/stix_transmission/delete_connector.py

```diff
@@ -0,0 +1,9 @@
+from stix_shifter_utils.modules.base.stix_transmission.base_delete_connector import BaseDeleteConnector
+
+
+class DeleteConnector(BaseDeleteConnector):
+    def __init__(self, api_client):
+        self.api_client = api_client
+
+    def delete_query_connection(self, search_id):
+        return {"success": True}
```

Comment:
looks like api doesn't allow deleting the search. so I don't think you need `delete_connector.py` at all.

Reply:
Yes @mdazam1942 Removed